components/PageLink.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link'
+import BackToButton from './BackToButton'
+
+interface PageLinkProps {
+  lookingForText: string
+  linkText: string
+  accessText: string
+  href: string
+  dataCy: string
+  buttonHref: string
+  buttonId: string
+  buttonLinkText: string
+  refPageAA: string
+  dashId: string
+}
+const PageLink = ({
+  lookingForText,
+  linkText,
+  accessText,
+  href,
+  dataCy,
+  buttonHref,
+  buttonId,
+  buttonLinkText,
+  refPageAA,
+  dashId,
+}: PageLinkProps) => {
+  const linkID = linkText ? linkText.replace(/\s+/g, '') : linkText
```

Comment:
While I agree with removing the extra id props, I think this should still be something along the lines of:
```suggestion
    const linkID = linkText.replace(/\s+/g, '')
```
Which is pretty much the same as it was before but removes the null check (which isn't needed given `linkText` is a required attribute). The ternary here seems a bit unnecessary.

Reply:
I see now that you changed this because the jest test was failing, but the reason the jest test was failing is because the `has no a11y violations` test is trying to render `PageLink` without supplying any of its required props (such as linkText). Changing the render to be:
```
    <PageLink
      lookingForText="title"
      accessText="accessText"
      linkText="Link text"
      buttonLinkText="buttonLinkText"
    />
```
fixes the issue.